### PR TITLE
Add PyCon CA 2016 videos

### PR DESCRIPTION
**Pull request by redapple**
Schedule: https://2016.pycon.ca/en/schedule/

YouTube video links found on each (scraped) talk page

### CODE REVIEW

**Comment by redapple** [comment]
@logston , @Daniel-at-github , any feedback?

File: pycon-ca-2016/videos/afternoon-keynote.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "\n",
```
Comment (logston) [inline pycon-ca-2016/videos/afternoon-keynote.json]: Description please.

File: pycon-ca-2016/videos/lightning-talks.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "\n",
```
Comment (logston) [inline pycon-ca-2016/videos/lightning-talks.json]: Description please.

File: pycon-ca-2016/videos/morning-keynote.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "\n",
```
Comment (logston) [inline pycon-ca-2016/videos/morning-keynote.json]: Description please.

File: pycon-ca-2016/videos/welcome-morning-keynote.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "\n",
```
Comment (logston) [inline pycon-ca-2016/videos/welcome-morning-keynote.json]: Description please.

**Comment by redapple** [comment]
@logston , descriptions updated. Will that do?

**Comment by logston** [comment]
Great. Thanks.

**Pull request merged**